packages/sourcecred/sharness/load_test_instance.t

```diff
@@ -19,15 +19,18 @@ export GIT_ATTR_NOSYSTEM=1
 # shellcheck disable=SC1091
 . ./sharness.sh
 
+toplevel="$(git -C "$(dirname "$0")" rev-parse --show-toplevel)"
+
+. $toplevel/scripts/monorepo_vars.sh
+
```

Comment:
This is a nuance with Lerna that I honestly don't completely understand. Inside a package, lerna will not return its
subpath.
Here's a fix:
```suggestion
cd "${toplevel}" || exit 1
. $toplevel/scripts/monorepo_vars.sh
cd "${CORE_PATH}" || exit 1

```